### PR TITLE
Add suspend/resume support to ef1e_tp and synchronization between ef1e_tp and serdes driver

### PR DESCRIPTION
**Pull request by phreer**

### CODE REVIEW

File: drivers/gpu/drm/bridge/ti/fpd_dp_ser_drv.c [hunk 1]
```diff
@@ -1789,26 +1845,38 @@ static int intel_get_i2c_bus_id(int adapter_id, char *adapter_bdf, int bdf_len)
...
 	fpd_dp_ser_set_up_mcu(fpd_dp_priv->priv_dp_client[0]);
 
+	fpd_dp_ser_set_ready(true);
+
+	fpd_dp_ser_unlock_global();
+
 	if (!fpd_dp_priv->priv_dp_client[2])
 		fpd_dp_priv->priv_dp_client[2] = i2c_new_dummy_device(fpd_dp_priv->i2c_adap, fpd_dp_i2c_board_info[2].addr);
 
+	msleep(500);
```
Comment (ljia5) [inline drivers/gpu/drm/bridge/ti/fpd_dp_ser_drv.c:1876]: Delete sleep 500 ms

Comment (phreer) [reply drivers/gpu/drm/bridge/ti/fpd_dp_ser_drv.c]: Resolved with the new version.

File: drivers/gpu/drm/bridge/ti/fpd_dp_ser_drv.c [hunk 2]
```diff
@@ -579,13 +632,15 @@ int fpd_dp_ser_set_dp_config(struct i2c_client *client)
...
 	fpd_dp_ser_write_reg(client, 0x49, 0x0);
-	pr_debug("[FPD_DP] Pull HPD High to start link training\n");
+	fpd_dp_ser_debug("[FPD_DP] Pull HPD High to start link training\n");
 	fpd_dp_ser_write_reg(client, 0x4a, 0x0);
 	fpd_dp_ser_write_reg(client, 0x4b, 0x1);
 	fpd_dp_ser_write_reg(client, 0x4c, 0x0);
 	fpd_dp_ser_write_reg(client, 0x4d, 0x0);
 	fpd_dp_ser_write_reg(client, 0x4e, 0x0);
 
+	/* Allow time after HPD is pulled high for the source to train and provide video */
```
[the remark this answers is not in the document]
Comment (phreer) [reply drivers/gpu/drm/bridge/ti/fpd_dp_ser_drv.c]: Resolved with the new version.

**Review by ljia5 — APPROVED** [review]

**Review by feijiang1 — APPROVED** [review]
LGTM

**Pull request merged**